ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md

```diff
@@ -1,4 +1,4 @@
-# PAS Explanatory Report for Open Grid Forum (OGF) Data Format Description Language (DFDL)
+# PAS Explanatory Report for Open Grid Forum (OGF) Data Format Description Language (DFDL) Transposition
```

Comment:
Drop 'Transposition'

Reply:
Transposition is the term for taking a PAS submission and flipping it into an ISO/IEC standard. 
Was added by ISO/IEC reviewer.

Reply:
Ah ok